Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -120,6 +113,54 @@ - (void)viewDidLoad
     [self refreshData];
 }
 
+- (void)setupNavigationItems
+{
+    self.titleButton = [UIButton buttonWithType:UIButtonTypeSystem];
+    [self.titleButton addTarget:self action:@selector(changeGroup:) forControlEvents:UIControlEventTouchUpInside];
+    self.titleButton.titleLabel.lineBreakMode = NSLineBreakByTruncatingMiddle;
+    self.titleButton.titleLabel.textAlignment = NSTextAlignmentCenter;
+    self.titleButton.titleLabel.numberOfLines = 1;
+
+    self.titleTipButton = [UIButton buttonWithType:UIButtonTypeSystem];
+    [self.titleTipButton addTarget:self action:@selector(changeGroup:) forControlEvents:UIControlEventTouchUpInside];
+    self.titleTipButton.titleLabel.textAlignment = NSTextAlignmentCenter;
+    self.titleTipButton.titleLabel.lineBreakMode = NSLineBreakByTruncatingMiddle;
+    self.titleTipButton.titleLabel.numberOfLines = 1;
+    NSString *localizedOptionHint = NSLocalizedString(@"Tap here to change", "Tip for tapping media picker title to change the group.");
+    NSString *callForAction = [NSString stringWithFormat:@"%@ %@",localizedOptionHint, ArrowDown];
```

Comment:
I think we should probably just put the arrow unicode in the localized string (and mention it in the comment), as I can imagine some localizations may want to move the position of the arrow (for example to the other side for RTL).